pacman/model/graphs/machine/machine_sdram_edge.py

```diff
@@ -0,0 +1,36 @@
+from pacman.model.graphs.common import EdgeTrafficType
+from pacman.model.graphs.machine import MachineEdge
+
+
+class SDRAMMachineEdge(MachineEdge):
+
+    __slots__ = [
+        "_sdram_size",
+        "_sdram_base_address"
+
+    ]
+
+    def __init__(self, pre_vertex, post_vertex, sdram_size, label):
+        MachineEdge.__init__(
+            self, pre_vertex, post_vertex, traffic_type=EdgeTrafficType.SDRAM,
+            label=label, traffic_weight=1)
+        self._sdram_size = sdram_size
+        self._sdram_base_address = None
+
+    @property
+    def sdram_size(self):
+        return self._sdram_size
+
+    @property
+    def sdram_base_address(self):
```

Comment:
We may instead want an SDRAMAllocation object, similar to the Placement object

Reply:
i dont agree. your adding complexity that isnt needed. the partition internally is doing said allocation based off its type. making a allocation obejct for this will just muddy the water

Reply:
p.s. sorry that this discussion is being done in a dead branch.